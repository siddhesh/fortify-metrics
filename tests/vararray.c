typedef __SIZE_TYPE__ size_t;

size_t
foo (size_t sz)
{
  int *buf[sz];
  return __builtin_object_size (buf, 0);
}
