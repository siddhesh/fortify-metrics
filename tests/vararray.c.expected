fmetrics:: 0:1
